packages/toast/test/toast.test.ts

```diff
@@ -56,14 +56,13 @@ describe('Toast', () => {
         it(`loads - [variant="${variant}"]`, async () => {
             const el = await fixture<Toast>(
                 html`
-                    <sp-toast variant=${variant} open>
+                    <sp-toast variant=${variant} open label="testLabel">
                         This toast is of the \`${variant}\` variant.
                     </sp-toast>
                 `
             );
-
             await elementUpdated(el);
-
+            await expect(el.getAttribute('label')).to.equal('testLabel');
```

Comment:
I couldn't check if the `sp-icon`'s label in this test since `sp-icon` isn't exposed in `sp-toast`

Reply:
Could you try getting it through the `shadowRoot`? So: 
```
el.shadowRoot.querySelector(
   sp-icon-checkmark-circle
) as HTMLElement;
```

I see that used in https://github.com/adobe/spectrum-web-components/blob/main/packages/tray/test/tray.test.ts#L105

Reply:
@sarahszhou Thank you, worked perfectly!